packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/XDD/Document.java

```diff
@@ -0,0 +1,60 @@
+package software.uncharted.terarium.hmiserver.models.xdd;
+
+import java.util.HashMap;
+import java.util.Hashtable;
+import java.util.List;
+
+import javax.json.bind.annotation.JsonbProperty;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+/**
+ * XDD Document representation
+ */
+public class Document {
+
+	@JsonbProperty("_gddid")
+	public String _gddid;
+
+	@JsonbProperty("title")
+  public String title;
+
+	// FIXME: can we have the field name without the "_" prefix?
+	@JsonbProperty("_abstract")
+	@JsonAlias("abstract")
+	public String _abstract;
+
+	@JsonbProperty("journal")
+  public String journal;
+
+	@JsonbProperty("type")
+  public String type;
+
+	@JsonbProperty("number")
+  public String number;
+
+	@JsonbProperty("pages")
+  public String pages;
+
+	@JsonbProperty("publisher")
+  public String publisher;
+
+	@JsonbProperty("volume")
+  public String volume;
+
+	@JsonbProperty("year")
+  public String year;
+
+	@JsonbProperty("link")
+  public List<HashMap<String, String>> link;
+
+	@JsonbProperty("author")
+  public List<HashMap<String, String>> author;
+
+	@JsonbProperty("identifier")
+  public List<HashMap<String, String>> identifier;
+
+	@JsonbProperty("known_terms")
+  public Hashtable<String, String[]> known_terms;
```

Comment:
variables should follow the regular expression '^[a-z][a-zA-Z0-9]*$'. please change to `knownTerms`

Reply:
Can you change Hashtable to Map and String[] to List<String>

Reply:
I know the difference between Map and Hashtable as well as the plain array versus List, but just curious what difference does it make to change the mentioned types in this context?

Reply:
done